inference-engine/src/mkldnn_plugin/nodes/mkldnn_non_max_suppression_node.cpp

```diff
@@ -147,14 +147,20 @@ void MKLDNNNonMaxSuppressionNode::prepareParams() {
         i.resize(num_classes);
 }
 
-void MKLDNNNonMaxSuppressionNode::createPrimitive() {
-    if (inputShapesDefined()) {
-        prepareParams();
-        updateLastInputDims();
-    }
+bool MKLDNNNonMaxSuppressionNode::isExecutable() const {
+    return isDynamicNode() || !hasEmptyInputTensors();
```

Comment:
From my point of view better to `!hasEmptyInputTensors()` because we can add some additional check into base `isExecutable()` which will be not valid for `MKLDNNNonMaxSuppressionNode::isExecutable()`